bench/README.md

```diff
@@ -65,57 +97,204 @@ And open `http://localhost:9090/` to query and monitor the server-side metrics.
 
 To run read load tests
 
-```bash
-locust DCLReadUser
-```
+* Open the docker-compose.yml and add a new `command` for `master` and `worker`.  
+    <br> e.g.: for `master`
+    ```yml
+    command:
+      - "ReadModelLoadTest" 
+    ```
+    e.g.: for `worker`: 
+    ```yml
+    command: 
+      - "ReadModelLoadTest" 
+    ```
 
-Then you can open `http://localhost:8089/` and launch the tests from the browser.
+* Open the docker-compose.yml and add a new field `extra_hosts` for `master` and `worker` (**this item need only for the local pool**):<br>
+  e.g.: for `master`
+  ```yml
+  extra_hosts:
+    - "host.docker.internal:host-gateway"
+  ```
+  e.g.: for `worker`
+  ```yml
+  extra_hosts:
+    - "host.docker.internal:host-gateway"
+  ```
+  `<host.docker.internal:host-gateway>` - need to connecting to localhost from locust container.
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine.
 
-### Configuration
+To run write/read load tests
 
-Run options (DCLedger custom ones):
+* Open the docker-compose.yml and add a new field `extra_hosts` for `master` and `worker` (**this item need only for the local pool**):<br>
+  e.g.: for `master`
+  ```yml
+  extra_hosts:
+    - "host.docker.internal:host-gateway"
+  ```
+  e.g.: for `worker`
+  ```yml
+  extra_hosts:
+    - "host.docker.internal:host-gateway"
+  ```
+  `<host.docker.internal:host-gateway>` - need to connecting to localhost from locust container.
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine. `Number of users should be equal to number of workers for write tests.`
 
-* `--dcl-users`: number of users
-* `--dcl-spawn-rate` Rate to spawn users at (users per second)
-* `--dcl-hosts <comma-sepated-list>`: list of DCL nodes to target. Each user randomly picks one
-    E.g. for local ledger `http://localhost:26657,http://localhost:26659,http://localhost:26661,http://localhost:26663` will specify all the nodes.
-* `--dcl-rest-hosts <comma-sepated-list>`: list of DCL nodes to target. Each user randomly picks one
-    E.g. for local ledger `http://localhost:26640,http://localhost:26641,http://localhost:26642,http://localhost:26643` will specify all the nodes.
-* `--dcl-txn-file` path to a file with generated txns
+Then you can open `http://localhost:8089/` and launch the tests from the browser.
 
 Statistic options:
 
```

Comment:
It's not clear where to set these statistic params. For what command they are provided?
It looks like they should be put into docker-compose.yml for the locust command. Please mention this explicitly.

Reply:
Done! ✅